<?xml version="1.0"?>
<launch>

	<node pkg="nodelet" type="nodelet" name="cpc_nodelet"  args="manager"/>

	<node pkg="nodelet" type="nodelet" name="PcPublisher" args="load cob_point_cloud_publisher/PcPublisher cpc_nodelet">
		<remap from="image_xyz" to="sensor_fusion/shared/image_xyz"/>
		<remap from="image_color" to="sensor_fusion/shared/image_color"/>
		<remap from="image_confidence" to="sensor_fusion/shared/image_confidence"/>
	</node>

</launch>
